components/Layout.tsx

```diff
@@ -13,17 +16,34 @@ interface Props {
 }
 
 export default function Layout({ children, title = 'Impact' }: Props): JSX.Element {
+	const { setThemeType, themeType } = useContext(AppContext)
+
 	return (
 		<>
 			<Head>
 				<title>{title}</title>
 			</Head>
 			<ElevationScroll>
 				<AppBar position="sticky">
-					<Toolbar variant="dense">
-						<Typography variant="h4" component="h1" gutterBottom>
-							{title}
-						</Typography>
+					<Toolbar>
```

Comment:
I was going to implement the nav bar as it's own component. Again another example of something that should have its own PR.